pandas/core/reshape/merge.py

```diff
@@ -952,6 +975,51 @@ def _validate_specification(self):
         if len(self.right_on) != len(self.left_on):
             raise ValueError("len(right_on) must equal len(left_on)")
 
+    def _validate(self, validate):
+
+        # Check uniqueness of each
+        if self.left_index:
+            left_unique = not (self.orig_left.index.duplicated()).any()
+        else:
+            left_unique = MultiIndex.from_arrays(self.left_join_keys
+                                                 ).is_unique
+
+        if self.right_index:
+            right_unique = not (self.orig_right.index.duplicated()).any()
+        else:
+            right_unique = MultiIndex.from_arrays(self.right_join_keys
+                                                  ).is_unique
+
+        # Check valid arg
+        if validate not in ['one_to_one', '1:1',
```

Comment:
can also put this in the end as an else clause, then don't need to list all options here again